bark/benchmark/benchmark_result.py

```diff
@@ -0,0 +1,289 @@
+# Copyright (c) 2019 fortiss GmbH
+#
+# This software is released under the MIT License.
+# https://opensource.org/licenses/MIT
+
+import os
+import pickle
+import pandas as pd
+import logging
+import re
+import zipfile
+import math
+
+logging.basicConfig()
+logging.getLogger().setLevel(logging.INFO)
+
+
+class BehaviorConfig:
+    def __init__(self, behavior_name, behavior, param_descriptions=None):
+        self.behavior_name = behavior_name
+        self.behavior = behavior
+        self.param_descriptions = param_descriptions or {}
+
+    def as_dict(self):
+        dct = {"behavior": self.behavior_name, **self.param_descriptions}
+        return dct
+
+    @staticmethod
+    def configs_from_dict(behavior_dict):
+        behavior_configs = []
+        for behavior_name, behavior in behavior_dict.items():
+            config = BehaviorConfig(behavior_name, behavior)
+            behavior_configs.append(config)
+        return behavior_configs
+
+# contains information for a single benchmark run
+class BenchmarkConfig:
+    def __init__(self, config_idx, behavior_config,
+                 scenario, scenario_idx, scenario_set_name):
+        self.config_idx = config_idx
```

Comment:
i plead for member variables to be like self._config_idx. Makes the separation more evident.